src/sentry/integrations/base.py

```diff
@@ -306,6 +306,15 @@ def get_default_identity(self):
         identity = Identity.objects.get(id=self.org_integration.default_auth_id)
         return identity
 
+    def get_stacktrace_link(self, repo, path, version):
+        """
+        Handle formatting and returning back the stack trace link if the client
+        request was successful.
+
+        If no file was found return `None`, and re-raise for non "Not Found" errors
+        """
+        pass
```

Comment:
@MeredithAnya we probably want to do a `NotImplementedError` error here I think

Reply:
Also putting in the repositories mixin instead of base